Source/WebCore/platform/graphics/FontDescription.cpp

```diff
@@ -83,7 +83,7 @@ static AtomString computeSpecializedChineseLocale()
 
 static AtomString& cachedSpecializedChineseLocale()
 {
-    static MainThreadNeverDestroyed<AtomString> specializedChineseLocale;
+    thread_local NeverDestroyed<AtomString> specializedChineseLocale;
     return specializedChineseLocale.get();
```

Comment:
We cannot use thread_local. We need to use ThreadSpecific etc. Also, having a single thread local slot here is anti-pattern in general. Why not having it in some places like WebCore's thread local data etc.?